### PR TITLE
Bug 1856353: Include installed version in operator hub item details

### PR DESCRIPTION
**Pull request by TheRealJon**
Update operator hub item details to include the installed version when it is not the latest version.
Update "Operator Version" detail to "Latest Version" to convey that it is not the actual installed
version.

image

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx [hunk 1]
```diff
@@ -225,12 +252,17 @@ export const OperatorHubPage = withFallback(
...
                   isList: true,
                   kind: referenceForModel(SubscriptionModel),
-                  prop: 'subscription',
+                  prop: 'subscriptions',
+                },
+                {
+                  isList: true,
+                  kind: referenceForModel(ClusterServiceVersionModel),
+                  prop: 'clusterServiceVersions',
                 },
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx:266]: This is a really expensive watch. CSVs contain a lot of data, and then they're copied across all namespaces. Are we able to wait until the user clicks the tile and get the single CSV we need to show the version?

Comment (TheRealJon) [reply frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx:266]: Yeah, we could feasibly do that.

**Comment by TheRealJon** [comment]
> This is a really expensive watch. CSVs contain a lot of data, and then they're copied across all namespaces. Are we able to wait until the user clicks the tile and get the single CSV we need to show the version?

@spadgett Looking into this, but I'm having issues with the details pane re-rendering very frequently. I'm trying to pinpoint the cause of that.

**Comment by TheRealJon** [comment]
/hold

Depends on https://github.com/openshift/console/pull/6456

**Pull request merged**